src/Custom-sample.h

```diff
@@ -21,6 +21,11 @@
     Here are some examples:
  */
 
+// --- Feature Flagging ---------------------------------------------------------
+
+#define FEATURE_ESP_P2P       1     // (1/0) enables the ESP Easy P2P protocol
```

Comment:
Please rename it to `FEATURE_ESPEASY_P2P` as I mentioned before.

Reply:
Sorry I missed that request.
I will do it now.

Reply:
rename done.